docs/modules/ROOT/pages/clc-config.adoc

```diff
@@ -27,108 +29,117 @@ Usage:
 clc config add [target] [source] [flags] [key=value...]
 ----
 
-Configuration keys:
+You can pass key=value pairs to add configuration items.
 
-[cols="1m,1a,1a,1"]
-|===
-|Key|Value|Default Value|Note
+.Configuration keys
+[%collapsible]
+====
+include::partial$configuration-items.adoc[]
+====
 
-|cluster.name
-|Any string
-|dev
-|
+.Global parameters
+[%collapsible]
+====
+include::partial$global-parameters.adoc[]
+====
 
-|cluster.address
-|HOST:PORT / HOST / :PORT
-|localhost:5701
-|
+Example output:
 
-|cluster.user
-|Any string
-|
-| Hazelcast Enterprise only
+[source,bash]
+----
+clc config add dev cluster.name=dev cluster.address=localhost:5701
+----
 
-|cluster.password
-|Any string
-|
-| Hazelcast Enterprise only
+== clc config import-cloud
 
-|cluster.discovery-token
-|Any string
-|
-|{hazelcast-cloud} only
+Imports configuration from a {hazelcast-cloud} CLI sample.
 
-|ssl.enabled
-|true / false
-|false
-|{hazelcast-cloud} or Hazelcast Enterprise
+In order to get the URL for the {hazelcast-cloud}, follow the steps below:
 
-|ssl.server
-|DOMAIN_NAME (must match SSL certificate)
-|
-|{hazelcast-cloud} (automatically set) or Hazelcast Enterprise
+1. On {hazelcast-cloud} console, visit:
++
+	Dashboard -> Connect Client -> CLI
 
-|ssl.skip-verify
-|true / false
-|false
-|{hazelcast-cloud} or Hazelcast Enterprise. Disables SSL host name and client-side verification. Never set to true in production.
+2. Copy the URL in box 2 and pass it as the second parameter.
+Make sure the text is quoted before running:
++
+[source,bash]
+----
+clc config import-cloud my-config "https://api..."
```

Comment:
would it be useful to include a suggestion rather than the ellipses? Something like:

clc config import-cloud my config "_URL copied from console_"

Reply:
Updated at: https://github.com/hazelcast/clc-docs/pull/15/commits/e30b3461375c3d99f63595286b6b229aa11c9021**